pyomo/contrib/doe/example/reactor_kinetics.py

```diff
@@ -79,159 +85,141 @@ def create_model(scena, const=False, control_time=None, control_val=None,
     controls = {}
```

Comment:
Yes. Do we remove the initialization for T in the example? Pyomo.DOE can do this with the designvariable object. I keep all of them so that it can run with the `parmest` option.